src/spec/doc/core-operators.adoc

```diff
@@ -579,7 +579,7 @@ it will return null instead of throwing a `NullPointerException`:
 ----
 include::../test/OperatorsTest.groovy[tags=spreaddot_nullsafe,indent=0]
 ----
-<1> build a list for which of of the elements is `null`
```

Comment:
this is in the official documentation for groovy under **8.1. Spread operator** section. 

http://groovy-lang.org/operators.html